frontend/spec/features/product_prices_spec.rb

```diff
@@ -34,7 +34,7 @@
 
       it 'will not render the currency selector' do
         visit spree.product_path(product)
-        expect(current_path).to eq(spree.product_path(product))
+        expect(current_path).to have_current_path(spree.product_path(product))
```

Comment:
The expectation needs to be on `page`  -   `expect(page).to have_current_path(...)`